### ISSUE TITLE
Webhook when results are unblinded

### ISSUE THREAD

**Issue by heinrichreimer** [tira-io/tira#464]
We are now increasingly using TIRA to automatically run privacy-preserving data aggregation tasks (e.g., for the [AQL dashboard](https://aql-dashboard.web.webis.de/)).
One issue, however, is that a system that automatically submitted a run does not know when this run is finished, unblinded, etc.
My suggestion here would be to allow users to specify [webhook URLs](https://en.wikipedia.org/wiki/Webhook) that are called when the state of the run changes.
That webhook could then trigger external downstream processing.

**Comment by mam10eks** [comment]
Good idea, we wanted to be more verbose anyway, with automatic messages to the users when a software is finished, unblinded, etc.

Discourse itself already allows for web-hooks, so we could reuse this: https://meta.discourse.org/t/configure-webhooks-that-trigger-on-discourse-events-to-integrate-with-external-services/49045
